src/token/handler.ts

```diff
@@ -166,6 +166,10 @@ export class InitialSqlTokenHandler extends TokenHandler {
     this.connection.emit('databaseChange', token.newValue);
   }
 
+  onDatabaseMirroringPartner(token: DatabaseMirroringPartnerEnvChangeToken) {
+    this.connection.emit('databaseMirroringPartner', token.newValue);
+  }
+
```

Comment:
Does this mean that the database mirroring partner event is sent by the server when the initial SQL setup query is sent by `tedious`?

This is... surprising. 🤔 

The MS-TDS specification states that:

> Type 13 (Database Mirroring) is sent in response to a LOGIN7 message whenever connection is requested to a database that it is being served as primary in real-time log shipping. The ENVCHANGE stream reflects the name of the partner node of the database that is being log shipped.

I'm wondering if there's some other issue in the sequencing of how we handle events and which state the handling happens in. Because based just on the specification, this event should not be send by the server in a response to the initial sql query we send.

Reply:
@arthurschreiber You're actually 100% correct, I mistakenly had added the event emitters in each of them during debugging but hadn't removed them to determine if it was a specific handler or not.

I've removed the unnecessary event handler from the `InitialSqlTokenHandler` class and verified that the event is still handled correctly.

Thanks!